server/api/submission.js

```diff
@@ -71,7 +75,7 @@ const isWithinSubmissionWindow = async () => {
 exports.post = async (
   {
     body: {
-      category, contestId, description, height, name, url, width,
+      category, contestId, description, height, name, url, width, backgroundColor,
```

Comment:
One thing I thought of, should we add a validation for background color that it's one of the 3 values we have? Don't want someone submitting (e.g using postman) a value that is either a weird color or worse isn't a color.

Reply:
Are you thinking:
`ALTER TABLE entries ADD background_color enum('#000', '#FFF', '#4b91e3');`

We can also do full hex to be consistent

Reply:
Yeah, if it makes it easier to do a regex for it.

Another (probably a bit overengineered) is to have a `background_colors` table, so the `entries` table only accepts colors we have in there. Those colors could then be passed down on the `/api/submissions` respone so we wouldn't have to update the front-end and back-end with new values.

Reply:
Ok, switched to full hex just for consistency.